lib/board.rb

```diff
@@ -1,15 +1,18 @@
 module TicTacToe
-
   class Board 
-    attr_reader :nine_space_array
+    attr_reader :cells
 
-    def initialize 
-      @nine_space_array = [*'0'..'8']
+    def initialize(cells = [*'0'..'8'])
+      @cells = cells
     end
 
     def fill(position, marker)
-      @nine_space_array[position.to_i] = marker
-      @nine_space_array
+      cells[position.to_i] = marker
+      self
+    end
+
+    def board_size
```

Comment:
tiny nit-pick: The method name `board_size` is slightly redundant given you will be calling the method on the `Board` object. `board.size` seems to sound a little bit better than `board.board_size`